plugin/pkg/admission/serviceaccount/admission.go

```diff
@@ -292,10 +297,21 @@ func (s *serviceAccount) limitSecretReferences(serviceAccount *api.ServiceAccoun
 			continue
 		}
 		secretName := source.Secret.SecretName
-		if !referencedSecrets.Has(secretName) {
+		if !mountableSecrets.Has(secretName) {
```

Comment:
> Our current statement is that if you're going to use a secret and you have service accounts enabled, the service account should list that secret as allowed.

Correct. That is not yet enforced by default admission, but it should be as soon as possible. I didn't have the nerve to set LimitSecrets to true by default in the initial pull.